src/components/Select/Select.jsx

```diff
@@ -396,7 +396,7 @@ export default class Select extends React.Component {
       return customButton(this.onButtonClick, this.onSearchChange);
     }
     if (customButton) {
-      return customButton(this.onButtonClick);
+      return customButton(this.onButtonClick, null, this.state.isOpen);
```

Comment:
heya @karelm you made a really great point! is this sort of what you had in mind?

Reply:
Yes, that is exactly what I meant 🙂 seems like too many words for such a small amount of code 😂 great stuff there @gmzjuliana 🙌

Reply:
🌮 to you @karelm for pointing it out!